crates/astria-conductor/src/config.rs

```diff
@@ -11,6 +11,13 @@ pub fn get() -> Result<Config, figment::Error> {
     Config::from_environment("ASTRIA_CONDUCTOR_")
 }
 
+#[derive(Debug, Serialize, Deserialize)]
+pub enum CommitLevel {
+    SoftOnly,
+    FirmOnly,
+    SoftAndFirm,
+}
```

Comment:
the enum fields are `SoftOnly` and `FirmOnly` but the env file says "Soft" and "Firm" are the options; is that correct?

Reply:
The env file should say `SoftOnly` and `FirmOnly`
I'll get that fixed.